lib/challenge.js

```diff
@@ -135,18 +168,27 @@ function prompt7() {
  */
 
 //  define someNumber here
-var someNumber;
+var someNumber = Math.floor(Math.random() * 101);
```

Comment:
Make sure to use `let` or `const` instead of `var`. `var` is older and can cause scoping issues.